docs/running-rules.md

```diff
@@ -9,39 +9,44 @@ import MoreHelp from "/src/components/MoreHelp"
 
 ## Introduction
 
-Rules are specific patterns based on which Semgrep reports findings in code. These findings may help you to catch issues of security, performance, correctness, and other bugs. Rules are stored in open-source Semgrep Registry that enables you to scan code without need to write anything custom.
+Rules are specific patterns based on which Semgrep reports findings in code. These findings may help you to catch issues of security, performance, correctness, and other bugs. Rules are stored in an open-source [Semgrep Registry](https://github.com/returntocorp/semgrep-rules) that enables you to scan code without need to write anything custom. Use Semgrep rules in the following ways:
 
-Contribute to the registry by writing your own rules and adding them to the <a href="https://github.com/returntocorp/semgrep-rules" target="_blank">Semgrep rules repository</a>.
+- Use existing [Semgrep Registry rules](#running-semgrep-registry-rules) or add new rules to Semgrep Registry and run them.
```

Comment:
Yeah I know. :-) Was not sure here, because when we say different kinds of rules, we are only describing them on the basis of how they are set up (in YAML, registry etc.), but not their use (Python rules, C rules, frameworks etc.). Do you have any idea here?

```
 The list below covers different kinds of Semgrep rules:

- Use existing [Semgrep Registry rules](#running-semgrep-registry-rules) or add new rules to Semgrep Registry and run them.
- Local rules:
  - Create and run one-off [ephemeral rules](#ephemeral-rules) in the command line.
  - Create and run [YAML-defined rules in a file](#creating-and-using-yaml-defined-rules-file).
- Run your [local rules simultaneously with Semgrep Registry rules](#running-multiple-rules-simultaneously). 
```

Reply:
Current state:

```markdown
The list below covers different kinds of Semgrep rules:

- Existing [Semgrep Registry rules](#running-semgrep-registry-rules). See [Contributing to Semgrep rules](https://semgrep.dev/docs/contributing/contributing-to-semgrep-rules-repository/) to find out how to add new rules.
- Local rules:
  - One-off [ephemeral rules](#ephemeral-rules) used in the command line.
  - The [YAML-defined rules](#creating-and-using-yaml-defined-rules-file) in a file.
- Combination of [local rules with Semgrep Registry rules](#running-multiple-rules-simultaneously). 
```